e2e-tests/pkg/tests/sharded/trx.go

```diff
@@ -361,6 +361,7 @@ func (c *Cluster) checkTrxCollection(ctx context.Context, col string, bcp Backup
 	}
 	c.flushRouterConfig(ctx)
 
+	time.Sleep(time.Second * 60)
```

Comment:
😬

Reply:
Not cool, but it's the easiest way to avoid sporadic test failures as from time to time PITR doesn't catch up to the desired restore point(